src/components/Filters/Filters.jsx

```diff
@@ -8,8 +10,28 @@ const Filters = () => {
     videosState: { filterText },
     dispatchVideo,
   } = useVideos();
+
+  const { navigate } = useAuth();
```

Comment:
Just a question. You can get the `navigate()` from the `useNavigate()` as well. So is there any advantage of providing the navigate via `AuthProvider`?

Reply:
I have imported `useNavigate` only in `authContext` file. So, wherever I want to use it, I'll just import it using `useAuth`. This could just trip off one line, that's it.